Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfiguration.swift

```diff
@@ -23,6 +23,11 @@ class PaymentMethodConfig: Codable {
             .adyenVipps,
             .adyenAlipay,
             .adyenGiropay,
+            .adyenInterac,
+			.adyenPayTrail,
```

Comment:
(⊙.⊙)

Reply:
you cannot spot that on Xcode but I made a change that will solve it